integrations/anthropic/src/haystack_integrations/components/generators/anthropic/chat/chat_generator.py

```diff
@@ -95,13 +100,18 @@ def __init__(
             - `temperature`: The temperature to use for sampling.
             - `top_p`: The top_p value to use for nucleus sampling.
             - `top_k`: The top_k value to use for top-k sampling.
-
+        :param ignore_tools_thinking_messages: Anthropic's approach to tools (function calling) resolution involves a
+        "chain of thought" messages before returning the actual function names and parameters in a message. If
+        ignore_tools_thinking_messages is True, the generator will drop so-called thinking messages when tool
+        use is detected. See the Anthropic [tools](https://docs.anthropic.com/en/docs/tool-use#chain-of-thought-tool-use)
+        for more details.
```

Comment:
Probably this should be indented.
Let's also put `include ignore_tools_thinking_messages` and `True` inside backticks